kedro/framework/cli/starters.py

```diff
@@ -493,6 +500,17 @@ def _make_cookiecutter_args(
     return cookiecutter_args
 
 
+def fetch_template_based_on_add_ons(template_path, cookiecutter_args: dict[str, Any]):
```

Comment:
If Pyspark option is selected we need to switch the template to the `spaceflights-pyspark` starter.